biojava-structure/src/main/java/org/biojava/nbio/structure/align/ce/CECalculator.java

```diff
@@ -957,9 +955,7 @@ private void checkBestTraces( AFPChain afpChain,
 
 		// removing some loops that are run in orig CE
 		// and which did not do anything
-		if ( debug ){
-			checkPrintRmsdNew(traceMaxSize, winSize, ca1, ca2);
-		}
+		checkPrintRmsdNew(traceMaxSize, winSize, ca1, ca2);
```

Comment:
What about wrapping with `if(LOGGER.isDebugEnabled())` and then modifying `checkPrintRmsdNew` to use the `LOGGER.debug`?